File: src/domain/entities/MobileAppRelease.ts
export interface MobileAppRelease {
  versionName: string
  versionCode: number
  apkUrl: string
  apkPath: string
  notes: string
  forceUpdate: boolean
  updatedAt: Date
  updatedById: string
  updatedByName: string
}

export function isNewerAppRelease(
  remoteVersionCode: number,
  installedVersionCode: number,
): boolean {
  return remoteVersionCode > installedVersionCode
}

File: consorcio/lib/domain/entities/mobile_app_release.dart
class MobileAppRelease {
  const MobileAppRelease({
    required this.versionName,
    required this.versionCode,
    required this.apkUrl,
    required this.notes,
    required this.forceUpdate,
  });

  final String versionName;
  final int versionCode;
  final String apkUrl;
  final String notes;
  final bool forceUpdate;

  bool isNewerThan(int installedVersionCode) =>
      versionCode > installedVersionCode;
}
